pkgs/development/libraries/libuv/default.nix

```diff
@@ -69,6 +69,8 @@ stdenv.mkDerivation (finalAttrs: {
         "fs_event_watch_dir_recursive" "fs_event_watch_file"
         "fs_event_watch_file_current_dir" "fs_event_watch_file_exact_path"
         "process_priority" "udp_create_early_bad_bind"
+        # fail on macos < 10.15 (starting in libuv 1.47.0)
+        "fs_write_alotof_bufs_with_offset" "fs_write_multiple_bufs" "fs_read_bufs"
```

Comment:
Can these be disabled only on x86_64-darwin? The oldest version supported on aarch64-darwin is 11.0, which is new enough for libuv.